contrib/twitchtv/twirp/twirp.go

```diff
@@ -0,0 +1,129 @@
+package twirp
+
+import (
+	"context"
+	"math"
+	"net/http"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/twitchtv/twirp"
+)
+
+type contextKey int
+
+const (
+	twirpErrorKey contextKey = 0
+	httpSpanKey   contextKey = 1
```

Comment:
Why not `iota`? Types seem to stay the same: https://play.golang.org/p/4Z9-Y_hWmuU